doc/examples/datasets_train/datasets_train.py

```diff
@@ -86,6 +134,10 @@ def create_data_chunk(n, d, seed, include_label=False):
             shuffle=False,
             random_state=seed)
 
+        # NOTE: To make things more interesting and more like "real" tabular data,
+        # we're going to introduce column names, categorical columns, and turn a few
+        # randomly select cells into NULL values. Not all data is clean and purely numeric!
```

Comment:
Is this intentionally duplicated (in the header doc)?

Reply:
removed, thx